src/routing/http-routing-v1.md

```diff
@@ -103,6 +106,22 @@ Response limit: 100 providers
 
 Each object in the `Providers` list is a *read provider record*.
 
+### `GET /routing/v1/peers/{peer-id}`
+
+#### Path Parameters
+
+- `peer-id` is the [Peer ID](https://github.com/libp2p/specs/blob/master/peer-ids/peer-ids.md) to fetch peer records for.
```

Comment:
There are 3 string types that are "peer ids", two of them are legacy, see the end of:
https://github.com/libp2p/specs/blob/master/peer-ids/peer-ids.md#string-representation

@hacdias  Thoughts on dropping legacy and requiring CIDv1 with libp2p-key codec here? Makes implementation easy -- plug in CID library, extract Multihash, done. Don't worry about legacy.

If we don't do it, we introduce complexity that will be additional work for implementers who care, and bugs and bad UX  if they don't

Reply:
I'm happy with only allowing the CIDv1 version.